pages/standard/design.tsx

```diff
@@ -1,50 +1,17 @@
 import React from 'react';
-import HtanNavbar from '../../components/HtanNavbar';
-import Footer from '../../components/Footer';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Breadcrumb from 'react-bootstrap/Breadcrumb';
-import { GetServerSideProps, GetStaticProps } from 'next';
-import fetch from 'node-fetch';
-import { CmsData } from '../../types';
-import { WORDPRESS_BASE_URL } from '../../ApiUtil';
+import { GetStaticProps } from 'next';
 
-export interface DesignProps {
-    data: CmsData[];
-}
+import DataStandard, { DataStandardProps } from "../../components/DataStandard";
+import { getStaticContent } from '../../ApiUtil';
 
-function Design(data: DesignProps) {
+const Design: React.FunctionComponent<DataStandardProps> = props => {
     return (
-        <>
-            <HtanNavbar />
-            <Container>
-                <Row>
-                    <Breadcrumb className="mt-3">
-                        <Breadcrumb.Item href="/">Home</Breadcrumb.Item>
-                        <Breadcrumb.Item href="/standards">
-                            Data Standards
-                        </Breadcrumb.Item>
-                        <Breadcrumb.Item active>Clinical Data</Breadcrumb.Item>
```

Comment:
@inodb I think this should be `Design Principles` instead of `Clinical Data`, right? (already fixed below)

Reply:
yes - thanks!